style.css

```diff
@@ -416,22 +421,53 @@ form > textarea {
     overflow-x: hidden;
   }
 
+  .right-side {
+    display: none;
+    justify-content: space-between;
+    background-color: rgba(96, 112, 255, 0.9);
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 600px;
+    z-index: 100;
+  }
```

Comment:
- Kindly make sure that the mobile menu is according to Figma's design. Fix the background to the full length of the device, as you can see your mobile menu background is half the length.
- Kindly make sure the background of the mobile menu is blurred as per the Figma design.
- Kindly make sure all mobile menu options are to a bit right side according to Figma design. 

Figma design:
<img width="290" alt="Screenshot 2022-04-05 at 7 03 41 PM" src="https://user-images.githubusercontent.com/70269507/161798697-c48b4df7-6ee8-4bf7-8da4-ff1000d1109d.png">

Your design:
<img width="500" alt="Screenshot 2022-04-05 at 7 08 38 PM" src="https://user-images.githubusercontent.com/70269507/161798730-42fa59bc-61c6-44ea-987d-171a826b113c.png">